tests/config/dapr_tracing_config.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: tracingconfig
+spec:
+  tracing:
+    samplingRate: "1"
+    zipkin:
```

Comment:
Shouldn't we use OTEL instead of Zipkin? That would be the recommended path now that we support OTEL. We can use Zipkin as the backend, just configure it through the OTel collector.

Reply:
Or, higher level question... Do we need to deploy Zipkin? Or can we create an endpoint (even inside the test app itself) that simulates zipkin and that accepts the requests? This way we don't need to deploy another service.

Reply:
This was the easiest way to reproduce the bug. I think we will have more scenarios but having an endpoint to handle zipkin would be too much IMO - the code is much simpler if we can reuse it instead of reimplementing zipkin API.

When it comes to OTEL instead of Zipkin, I can look into it - but these are not exclusive. We need to expand the tracing test scenarios beyond this one. This is a first step.